interop/xds/custom_lb_test.go

```diff
@@ -66,7 +66,7 @@ func (s) TestCustomLB(t *testing.T) {
 			}
 			rpcBMD := md.Get("rpc-behavior")
 			if len(rpcBMD) != 1 {
-				errCh.Send(errors.New("only one value received for metadata key rpc-behavior"))
+				errCh.Send(errors.New("only one value should be received for metadata key rpc-behavior"))
```

Comment:
yes only to fix the error message, because it says "one value received" even though the `len(rpcBMD) != 1` check means that is not the case

not required to be a part of this PR but I think it's a typo that needs to be fixed?

Reply:
Sure, if we can have it as part of a separate PR, that would be nicer. That way, we can keep the discussion here in this PR to the implementation of `rpc-behavior` metadata.

Reply:
I've created PR #6614 for this
thanks